department-of-veterans-affairs/va.gov-cms
# VACMS-5713 - turn on animated gifs

## Description
Closes #5713

## Testing done
Visual

## QA steps
- As an administrator, go to `node/26856/edit` and add an animated gif via the Embedded image block in the main content block.
- Save the page, and visually verify the gif is animated on the page.

### Definition of Done


### Select Team for PR review


### Is this PR blocked by another PR?


### Does this PR need review from a Product Owner


### CMS user-facing annoucement

Is an announcement needed to let editors know of this change?
